lib/templates/plugin.js

```diff
@@ -1,3 +1,9 @@
-export default async function ({ router, store }) {
-  //
+function sampleFunction () {
```

Comment:
The idea for plugin injections is to inject a service bound to request context. For context-independent helpers one can use vue.mixin which is also faster.